spec/unit/provider/postgresql_conf/ruby_spec.rb

```diff
@@ -29,6 +30,28 @@
   it 'has a method exists?' do
     expect(provider).to respond_to(:exists?)
   end
+  context '#exists?' do
```

Comment:
I'd write it as:
```suggestion
  describe '#exists?' do
```

That it responds to `exists?` is a weaker test than the ones you're adding. It's also custom to use `describe` if you're describing a method.